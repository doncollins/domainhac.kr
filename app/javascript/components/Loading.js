import React from 'react'

export default function() {
    return (
        <div className="flex justify-center w-full">
            <div className="text-blue-500 text-3xl">
                <i className="fa fa-spin fa-circle-o-notch" />
            </div>
        </div>
    )
}