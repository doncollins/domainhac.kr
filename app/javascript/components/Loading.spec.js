import React from 'react'
import renderer from 'react-test-renderer';
import Loading from './Loading';

it('renders correctly', () => {
    const component = renderer.create(
        <Loading />
    )

    expect(component.toJSON()).toMatchSnapshot()
})