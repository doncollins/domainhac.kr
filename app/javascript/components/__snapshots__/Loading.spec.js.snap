// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="flex justify-center w-full"
>
  <div
    className="text-blue-500 text-3xl"
  >
    <i
      className="fa fa-spin fa-circle-o-notch"
    />
  </div>
</div>
`;
